src/tpl/register-confirm.html

```diff
@@ -3,8 +3,10 @@
         <div class="container">
             <div class="row">
                 <div class="my-5 col-md-8">
-
+                    
+                    {{^errorMsg}}
                     <h3>Confermare registrazione dell'URL nel database?</h3>
+                    {{/errorMsg}}
```

Comment:
Maybe we should show the error if the entry is duplicated.

Reply:
+1 on this, it can be a nice hint. Also, this is public info so we got nothing to hide :smiley:

Reply:
error is already there, this template inherit the default layout where the error section is placed: [see here](https://github.com/italia/developers-italia-onboarding/blob/master/public/index.html#L117)

Reply:
@sebbalex sorry, I missed that :+1: